src/main/java/ai/lucidtech/las/sdk/Client.java

```diff
@@ -256,6 +256,7 @@ public JSONObject updateDocument(
      * @return Batch handle id and pre-signed upload url
      * @throws IOException General IOException
      * @throws APIException Raised when API returns an erroneous status code
+     * @throws MissingAccessTokenException Raised if access token cannot be obtained
```

Comment:
This definition is more clear, could you change it for the other endpoint?

Reply:
Yes